dist/style.css

```diff
@@ -0,0 +1,43 @@
+.hero {
+    background: url(img/restaurant.jpg) no-repeat center center fixed;
+    background-color: #4f3222;
+    background-blend-mode: screen;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+}
+
+.center, .center-column, .top, .right, .bottom, .left {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.center-column {
+    flex-direction: column;
+}
+
+.top { align-items: flex-start;}
+.right { justify-content: flex-end;}
+.bottom { align-items: flex-end;}
+.left { justify-content: flex-start;}
+
+.single-spaced, .single-spaced * {
+    line-height: 1;
+}
+
+
+#a{ grid-area: a;}
+#b{ grid-area: b;}
+#c{ grid-area: c;}
+#d{ grid-area: d;}
+#e{ grid-area: e;}
+
+#grid {
+    display: grid;
+    grid-template-areas:
+        "a a a a"
+        "b c d e";
+    grid-row-gap: 1rem;
+}
```

Comment:
Maybe you can add a custom rule to the `.navbar-item` to make it clear that they are clickable, otherwise, it is difficult to tell that those elements are clickable:

    .navbar-item {
        cursor: pointer;
    }

Even some [microinteraction](https://www.nngroup.com/articles/microinteractions/) on `hover` or after it was click to have a better UX.